app/controllers/pages_controller.rb

```diff
@@ -5,11 +5,13 @@ def home
       @timeline = true
 
       if params[:before]
-        @before = Nweet.find_by(url_digest: params[:before])
-        @feed_items = current_user.timeline.where('did_at < ?', @before.did_at).limit(10)
-        render partial: 'nweets/nweet', collection: @feed_items
+        date = Time.at(params[:before].to_i)
```

Comment:
```suggestion
        date = Time.zone.at(params[:before].to_i)
```
User.did_at のようにdatetime型のカラムはRailsでは`ActiveSupport::TimeWithZone`クラスとして扱います（Nuitaではデフォルトのcreated_at等も含めほとんどの時刻関連がこれです）。
`Time.at()`はTime型を、`Time.zone.at()`は`TimeWithZone`型を返すので、後者を使ったほうがよいです。